site/docs/v1/tech/oauth/integrations/salesforce.adoc

```diff
@@ -0,0 +1,213 @@
+
+[cols="1,2,1"]
+|===
+| Salesforce field | FusionAuth field | Sample value
+
+| [field]#Provider Type#
+| `Open ID Connect`
+| {nbsp}
+
+| [field]#Name#
+| FusionAuth
+| {nbsp}
+
+| [field]#URL Suffix#
+| FusionAuth
+| {nbsp}
+
+| [field]#Consumer Key#
+| Your FusionAuth application [field]#Client Id#
+| {nbsp}
+
+| [field]#Consumer Secret#
+| Your FusionAuth application [field]#Client Secret#
+| {nbsp}
+
+| [field]#Authorize Endpoint URL#
+| Your FusionAuth application [field]#OAuth IdP login URL# until the question mark (`?`)
+| `\https://your.fusionauth.application.url/oauth2/authorize`
+
+| [field]#Token Endpoint URL#
+| Your FusionAuth application [field]#Token endpoint#
+| `\https://your.fusionauth.application.url/oauth2/token`
+
+| [field]#User Info Endpoint URL#
+| Your FusionAuth application [field]#Userinfo endpoint#
+| `\https://your.fusionauth.application.url/oauth2/userinfo`
+
+| [field]#Token Issuer#
+| Your FusionAuth application [field]#Userinfo issuer#
+| `\https://your.fusionauth.application.url`
+
+| [field]#Default Scopes#
+| `openid email`
+| {nbsp}
+
+| [field]#Send access token in header#
+| Checked
+| {nbsp}
+
+| [field]#Send client credentials in header#
+| Unchecked
+| {nbsp}
+
+| [field]#Include Consumer Secret in SOAP API Responses#
+| Unchecked
+| {nbsp}
+
+| [field]#Custom Logout URL#
+| Your FusionAuth application [field]#Logout Url#
+| `\https://your.fusionauth.application.url/oauth2/logout?client_id=Application-Client-Id`
+|===
+
+In [field]#Registration Handler#, click the icon:search[type=fas] magnifying glass button to open a window with the existing Apex Classes in your Salesforce organization. There, click `FusionAuthRegHandler` to select it. Click the other icon:search[type=fas] magnifying glass button in [field]#Execute Registration As# and select the user that will be responsible for executing the registration (you can select your own user here).
+
+Click [uielement]#Save# to finish configuring the provider. Scroll down to the [uielement]#Salesforce Configuration# section and open the address from [uielement]#Test-Only Initialization URL# in an incognito window.
+
+image::oauth/integrations/salesforce/auth-provider-values.png[Copy Test-Only Initialization URL from the Authentication Provider.,width=1200]
+
+After logging in with your FusionAuth credentials, you should be redirected to an XML file with the user details that Salesforce will receive from FusionAuth, like the example below.
+
+```xml
+<user>
+    <full_name>Your Full Name</full_name>
+    <provider>Open ID Connect</provider>
+    <org_id>000000000000000</org_id>
+    <last_name>Your Last Name</last_name>
+    <id>00000000-0000-0000-0000-000000000000</id>
+    <portal_id>000000000000000</portal_id>
+    <first_name>Your First Name</first_name>
+    <email>Your Email</email>
+</user>
+```
+
+[NOTE.note]
+====
+If you are not seeing these values, click [uielement]#Edit# in the Salesforce `Auth. Providers` page and double-check the values.
```

Comment:
Would this formatting be more consistent with the other places we reference page names?
```suggestion
If you are not seeing these values, click [uielement]#Edit# in the Salesforce [breadcrumb]#Auth. Providers# page and double-check the values.
```